apiserver/params/params.go

```diff
@@ -498,6 +498,20 @@ type ApplicationUnitParams struct {
 	Data       map[string]interface{} `json:"data"`
 }
 
+// UpdateApplicationServiceArgs holds the parameters for
+// updating application services.
+type UpdateApplicationServiceArgs struct {
+	Args []ApplicationServiceParams `json:"args"`
+}
+
+// ApplicationServiceParams holds parameters used to update
+// an application's service definition for the cloud.
+type ApplicationServiceParams struct {
```

Comment:
The names for these types are pretty confusing, especially in the client code. `ApplicationServiceArg` would be clearer for this one, even though it's composed of multiple values.

Reply:
ok